src/main/webapp/js/controllers/TransferRequestController.js

```diff
@@ -17,14 +18,10 @@ gyfp.controller("TransferRequestController", ["$scope", "$modal", "$routeParams"
     $scope.authenticated = false;
 
     // Watch for change in authentication state
-    $scope.$watch(authService.isAuthenticated, function(isAuthenticated) {
+    $scope.$on("AuthenticationService.AuthenticationChanged", function(event, isAuthenticated) {
         $scope.authenticated = isAuthenticated;
         if (isAuthenticated) {
             $scope.loadRequest();
-        } else {
-            // If we are not authenticated, attempt to authenticate using
-            // existing credentials to prevent having to click the login button
-            authService.checkAuth();
```

Comment:
It was essentially causing an infinite loop, because checkAuth would cause
another failed auth broadcast, which would in turn cause another checkAuth,
causing ...

It needs to be reworked and added back at some point, but I'm not exactly
how/where yet.

On Thu, Feb 12, 2015 at 8:54 AM, Reggie Pierce <notifications@github.com>
wrote:

> In src/main/webapp/js/controllers/TransferRequestController.js
> <https://github.com/RITHonorsTechCommittee/gimme-your-files-please/pull/43#discussion_r24581471>
> :
>
> >          $scope.authenticated = isAuthenticated;
> >          if (isAuthenticated) {
> >              $scope.loadRequest();
> > -        } else {
> > -            // If we are not authenticated, attempt to authenticate using
> > -            // existing credentials to prevent having to click the login button
> > -            authService.checkAuth();
>
> Just out of curiosity, why was this removed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RITHonorsTechCommittee/gimme-your-files-please/pull/43/files#r24581471>
> .
>